src/libs/theme.js

```diff
@@ -266,6 +266,21 @@ export const Styles = {
       justifyContent: "left",
       alignItems: "center",
     },
+    ID_CELL: {
+      width: "5%",
+      margin: "0 1%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    INSTITUTION_CELL: {
+      maxWidth: "35%",
+      minWidth: "35%",
+      margin: "0 1%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
```

Comment:
I have mixed opinions on this being here. I can go either way, so this shouldn't hold up the PR. My thought is that if something is very specific to a single page, then the styling should be there and merged into what we get from the generic theme which effectively overrides or supersedes any conflicting properties. This seems like one of those cases where the institution cell is specific to the manage institutions page. However, if you foresee other pages making use of this style, then I can see the point in adding it here.

Reply:
The downside of putting every possible styling into this theme is that it ends up being a hodge-podge of styles that become very difficult to use.